test/unit/integrations/payu_in/payu_in_notification_test.rb

```diff
@@ -30,10 +30,6 @@ def test_accessors
     assert_equal true, @payu.checksum_ok?
   end
 
-  def test_compositions
-    assert_equal '10.00', @payu.gross
```

Comment:
this should still be valid no? Should we update all other `test_compositions` to also use `gross`?